CMakeLists.txt

```diff
@@ -60,11 +60,11 @@ if(LIBCELLML_COVERAGE OR LIBCELLML_MEMCHECK)
   set(LIBCELLML_UNIT_TESTS ON CACHE BOOL ${_PARAM_ANNOTATION} FORCE)
 endif()
 
-# WARNINGS_AS_ERRORS ==> LIBCELLML_WARNINGS_AS_ERRORS -- Note: This excludes third party code, where warnings are never treated as errors.
+# TWAE ==> LIBCELLML_WARNINGS_AS_ERRORS -- Note: This excludes third party code, where warnings are never treated as errors.
```

Comment:
Should be `LIBCELLML_TREAT_WARNINGS_AS_ERRORS` in this comment... alas for the nitpicking!